personalcapital.js

```diff
@@ -108,6 +124,34 @@ function updateBlockcypherBalancePromise(account, symbol, smallestUnit) {
     });
 }
 
+var erc20Dictionary = {};
+function getErc20BalancePromise(account, symbol) {
+    var ethereumAddress = account.description.slice(6);
+    return new Promise(function(resolve, reject) {
+        if (erc20Dictionary[ethereumAddress]) {
+            resolve(erc20Dictionary[ethereumAddress]);
+        }
+        reject();
```

Comment:
This is a neat solution but its hard to understand at first glance. It would probably be easier for future maintainers reading this code to just call `getJSON().then(...` and following the code top to bottom instead of rejecting the promise immediately and having to jump around figuring out what path we are taking.

Reply:
I believe I had a reason for not doing that originally but I don't remember.